packages/editor/src/editor/palette/Palette.tsx

```diff
@@ -14,7 +14,9 @@ export const Palette = ({ sections, directCreate }: PaletteProps) => {
     <Flex direction='column' className='palette' gap={3}>
       <SearchInput placeholder='Search...' value={searchTerm} onChange={setSearchTerm} />
       {Object.entries(sections).map(([section, sectionItems]) => {
-        const filteredItems = sectionItems.filter(item => item.name.toLowerCase().includes(searchTerm.toLowerCase()));
+        const filteredItems = sectionItems
+          .filter(item => item.name !== 'DataTableColumn')
+          .filter(item => item.name.toLowerCase().includes(searchTerm.toLowerCase()));
```

Comment:
Not sure if you understand what I mean...
I think this should not be filtered by the palette components. Or at least not by the item name.
What if this item name changes one day (what I think is not too unrealistic)? -> bug

Why not add a new category "hidden"? then the filtering here would make much more sense.